### ISSUE TITLE
Proposal: Add left/right keyboarding behavior to Expander

### ISSUE THREAD

**Issue by kat-y** [microsoft/microsoft-ui-xaml#3679]
# Proposal: Add left/right keyboarding behavior to Expander

## Summary

When focus is on the Header of the Expander, a user should be able to hit Space, Enter, or Right arrow to expand it [(see Expander spec here](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/kat-y/expander/active/Expander/Expander.md)). After this, hitting Space, Enter, or Left arrow should collapse it. Currently Expander responds to Space and Enter, but not right/left arrow keys - see https://github.com/microsoft/microsoft-ui-xaml/tree/master/dev/Expander 

## Rationale

* Aligns with the behavior in TreeView for expand/collapsing with right/left arrows
* recommended for accessibility for the open/closing behavior

## Scope

| Capability | Priority |
| :---------- | :------- |
| Allow users to use right arrow to expand the Expander | Must |
| Allow users to use left arrow to collapse the Expander | Must |

## Open Questions

Is there a RTL consideration for assigning right/left as expanding/collapsing? This has been brought up with PMs knowledgeable about RTL use and I'll update here with the response.

**Comment by chingucoding** [comment]
Given that the layout would be inverted in RtL, so should the left/right arrow behaviors. That is also done by controls such as ColorPicker.

Another thing: How would navigating using Narrators Itemmode and arrow keys work with that? Would navigating to the next item (itemmode + right arrow key) open the expander?
